src/constraint/constrSys.ml

```diff
@@ -294,6 +292,7 @@ struct
               f get set
             )
         in
-        Some f'
+        f'
+        ) (S.system x)
```

Comment:
I think usually we have the lambda body indented by one more level in calls like this.